### PR TITLE
Proposal Feedback: Comparison of Splunk and Nagios Devop monitoring tools and their use cases

### PR DESCRIPTION
**Pull request by Renstrom**
# Feedback Proposal: Comparison of Splunk and Nagios Devop monitoring tools and their use cases (#1021)

## Members

Yuxin Meng [@yuxinm](https://github.com/yuxinmmmm) email: yuxinm@kth.se
Anders Renström [@renstr](https://github.com/Renstrom) email: renstr@kth.se

## Proposal

Want to provide feedback for the above essay. [#1021](https://github.com/KTH/devops-course/pull/1021).

### CODE REVIEW

File: contributions/feedback/renstr-yuxinm/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Feedback Proposal: Comparison of Splunk and Nagios Devop monitoring tools and ther use cases (#1021)
+
+## Members
+
+Yuxin Meng [@yuxinm](https://github.com/yuxinmmmm)
+Anders Renström [@renstr](https://github.com/Renstrom)
```
Comment (khesoem) [inline contributions/feedback/renstr-yuxinm/README.md]: Please write your full KTH email.

**Review by khesoem — CHANGES_REQUESTED** [review]

**Comment by khesoem** [comment]
Please add a comment to #1021 and mention that you will give feedback on it.

File: contributions/feedback/renstr-yuxinm/README.md (hunk 1 above)
Comment (Renstrom) [reply contributions/feedback/renstr-yuxinm/README.md]: Done

**Comment by carinawic** [comment]
I have submitted the essay now as a PR and looking forward to receive your feedback!

The PR is number #1338

**Comment by carinawic** [comment]
I have written a comment "Response to feedback" describing how we altered our report thanks to your feedback in #1338 . Thank you!

**Pull request merged**